### PR TITLE
fix(popover): correct left/right tips for RTL, increase VRT coverage

### PR DESCRIPTION
**Pull request by mdt2**
## Description

Fixes an issue in Popover where Left and Right tips were mirrored incorrectly for RTL. This is simliar to some [work done on Tooltip](https://github.com/adobe/spectrum-css/pull/2736). Note that the Left and Right placement for the tips are always on the LTR left and right, even if RTL is turned on. Start and End, however, are adjusted based on the text direction.

Also included Storybook enhancements including:
- Show all placement options (previously Start and End placements weren't included in the Storybook controls)
- Increase Chromatic VRT coverage for the tip variant

## How and where has this been tested?

Please tag yourself on the tests you've marked complete to confirm the tests have been run by someone other than the author.

### Validation steps 

- [x] VRTs passing @mdt2 
- [x] Check Popover With Tip's new [testing preview](https://pr-2753--spectrum-css.netlify.app/preview/?path=/story/components-popover--with-tip&args=isOpen:!true&globals=testingPreview:!true) in Storybook. All the tips should look correctly positioned in LTR @jawinn
- [x] Now check RTL, tips should be correctly positioned
- [x] All placement options are shown in the Storybook controls

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:

- [ ] The pages render correctly, are accessible, and are responsive.

2. If components have been modified, VRTs have been run on this branch:

- [ ] VRTs have been run and looked at.
- [ ] Any VRT changes have been accepted (by reviewer and/or PR author), or there are no changes.

## Screenshots

Before
before-popover

After
after-popover

## To-do list

- [x] I have read the [contribution guidelines](/.github/CONTRIBUTING.md).
- [x] I have updated relevant storybook stories and templates.
- [x] If my change impacts **documentation**, I have updated the documentation accordingly.
- [ ] ✨ This pull request is ready to merge. ✨

### CODE REVIEW

File: components/popover/stories/popover.stories.js [hunk 1]
```diff
@@ -113,21 +127,71 @@ export default {
...
 };
 
+
+const ChromaticTipPlacementVariants = (args) => html`
+	${placementOptions.map(option => {
+		const optionDescription = () => {
+			if (option.startsWith("start") || option.startsWith("end"))
+				return "Changes side with text direction (like a logical property)";
+			if (option.startsWith("left") || option.startsWith("right"))
+				return "Text direction does not effect the position";
```
Comment (jawinn) [inline components/popover/stories/popover.stories.js:150]:
```suggestion
				return "Text direction does not affect the position";
```

**Review by jawinn — APPROVED** [review]
Nice work fixing this and adding the Storybook improvements. The Chromatic-only view with descriptions makes it much easier to understand which ones should be flipping.

I only have one minor grammar suggestion.

File: .changeset/three-geese-relax.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/popover": patch
+---
+
+fix(popover): correct left/right tips for RTL
```
[the remark this answers is not in the document]
Comment (mdt2) [reply .changeset/three-geese-relax.md:5]:
```suggestion
fix(popover): correct left/right tips for RTL

Includes exposing Start and End tip placement variants in Storybook, and increased VRT coverage for Chromatic.
```

File: components/popover/index.css [hunk 3]
```diff
@@ -150,7 +150,8 @@ governing permissions and limitations under the License.
 	/* spacing to include tip in calculation of offset from source */
 	&.spectrum-Popover--withTip {
 		/* tip size minus where it overlaps with popover border */
-		margin-inline-start: calc(var(--mod-popover-pointer-width, var(--spectrum-popover-pointer-width)) - var(--mod-popover-border-width, var(--spectrum-popover-border-width)));
+		/* stylelint-disable-next-line csstools/use-logical -- intentional, right stays on the same side even for RTL */
```
Comment (castastrophe) [inline components/popover/index.css:153]: Thanks for adding this detailed comment!  Very helpful and ensures we don't accidentally "correct" it later.

File: components/popover/stories/template.js [hunk 4]
```diff
@@ -159,3 +171,39 @@ export const Template = ({
 		</div>
 	`;
 };
+
+export const SourcelessTemplate = ({
```
Comment (castastrophe) [inline components/popover/stories/template.js:175]: Can you add some more context in a comment here about why this additional template is needed?

**Review by castastrophe — CHANGES_REQUESTED** [review]
This is looking good, just a question about the goals of the sourceless template. I'm wondering if we have to update the class list or add new features, is it going to be hard to remember to update that in both templates? Is there a way to have 1 be a "source of truth" without needing the duplication?

File: components/popover/stories/template.js (hunk 4 above)
Comment (mdt2) [reply components/popover/stories/template.js]: Great callout on this! Turns out all I had to do was pass a null trigger where I didn't want a source button to show up (for Chromatic), so I've removed this code 👍

**Review by castastrophe — APPROVED** [review]

**Pull request merged**